automated install/basic-install.sh

```diff
@@ -83,6 +83,7 @@ PI_HOLE_INSTALL_DIR="/opt/pihole"
 PI_HOLE_CONFIG_DIR="/etc/pihole"
 PI_HOLE_BIN_DIR="/usr/local/bin"
 FTL_CONFIG_FILE="${PI_HOLE_CONFIG_DIR}/pihole-FTL.conf"
+FTL_PID_FILE="/run/pihole/FTL.pid"
```

Comment:
You basically hard-code the path here. However, we have an option in `pihole-FTL.conf` to change the path.
https://docs.pi-hole.net/ftldns/configfile/#file_PIDFILE

In any case (even if just changing the default path without removing the option to change it), this PR needs a change in https://github.com/pi-hole/FTL as well.

Reply:
> You basically hard-code the path here. However, we have an option in pihole-FTL.conf to change the path.

That's what I'm doing. The code later is changing the path via the pihole-FTL.conf file.

> this PR needs a change in https://github.com/pi-hole/FTL as well.

Will do in case the PR is accepted. [Here](https://github.com/pi-hole/FTL/blob/e938e27c2e6adb10980293695911828bedd22836/src/config.c#L339) right?

Reply:
> The code later is changing the path via the pihole-FTL.conf file.

You write the path to `pihole-FTL.conf` but you never check if users made change to that file an source `FTL_PID_FILE` again.